lib/tools/adb-emu-commands.js

```diff
@@ -38,9 +38,15 @@ emuMethods.fingerprint = async function (fingerprintId, udid = undefined) {
   if (udid) {
     await this.setDeviceId(udid);
   }
+  this.resetTelnetAuthToken();
   await this.adbExec(['emu', 'finger', 'touch', fingerprintId]);
 };
 
+emuMethods.rotate = async function () {
```

Comment:
when users set the deviceOrientation capability or even during tests to verify their apps are rotating(or not) according to what the want to give as user experience.